examples/persons/src/EditPerson.re

```diff
@@ -11,6 +11,34 @@ module EditPersonMutation = [%graphql
   |}
 ];
 
+module OptimisticResponse = {
+  type t = {
+    .
+    "__typename": string,
+    "updatePerson": {
+      .
+      "__typename": string,
+      "age": int,
+      "id": string,
+      "name": string,
+    },
+  };
```

Comment:
This should be compatible with all versions of bucklescript, but can be changed to only work with bucklescript >= 7 if that's preferred.

Reply:
alright!